database/neo4j/neo4j.go

```diff
@@ -0,0 +1,202 @@
+package neo4j
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	neturl "net/url"
+	"sync/atomic"
+
+	"github.com/golang-migrate/migrate/v4/database"
+	"github.com/neo4j/neo4j-go-driver/neo4j"
+)
+
+func init() {
+	db := Neo4j{}
+	database.Register("bolt", &db)
+	database.Register("neo4j", &db)
+}
+
+var DefaultMigrationsLabel = "SchemaMigration"
+
+var (
+	ErrNilConfig = fmt.Errorf("no config")
+)
+
+type Config struct {
+	AuthToken       neo4j.AuthToken
+	URL             string
```

Comment:
Add a comment about not specifying auth in the `URL`

Reply:
Done